src/debug/daccess/CMakeLists.txt

```diff
@@ -63,4 +63,4 @@ if(CLR_CMAKE_PLATFORM_UNIX)
   )
 
   add_dependencies(daccess coreclr dactablerva_header)
-endif(CLR_CMAKE_PLATFORM_UNIX)
+endif(CLR_CMAKE_PLATFORM_UNIX AND NOT CLR_CROSS_COMPONENTS_BUILD)
```

Comment:
We cannot build cross-arch mscordaccore in Linux yet. But someday we may try to build mscordaccore. If these dependencies need for mscordaccore, it's better to remain dependencies, and remove coreclr dependency only.